tls/tests/test_remote.py

```diff
@@ -6,288 +6,294 @@
 from six import PY2
 
 from datadog_checks.base import ConfigurationError
-from datadog_checks.tls import TLSCheck
+from datadog_checks.tls.const import (
+    SERVICE_CHECK_CAN_CONNECT,
+    SERVICE_CHECK_EXPIRATION,
+    SERVICE_CHECK_VALIDATION,
+    SERVICE_CHECK_VERSION,
+)
+from datadog_checks.tls.tls_remote import TLSRemoteCheck
 
 
 def test_no_server(instance_remote_no_server):
-    c = TLSCheck('tls', {}, [instance_remote_no_server])
+    c = TLSRemoteCheck('tls', {}, [instance_remote_no_server])
```

Comment:
```suggestion
    c = TLSCheck('tls', {}, [instance_remote_no_server])
```
Why not using `TLSCheck` directly? So we make sure the test is going well from the `TLSCheck` constructor
(same for the other tests)

Reply:
Yeah, I was a bit in between of being explicit or not because it's an unit test but I think you're right